Adding downloading pdfs from a submission id

andrew8bit opened the pull request:
Advise on how to test - tested locally for now

Scott771 commented:
@andrew8bit want to clean up the example (the last example doesn't belong in the example file and I think that file is named with a ,py instead of .py) and merge this? Ideally you'd also have a test for this. Or can close it for now

andrew8bit commented:
> @andrew8bit want to clean up the example (the last example doesn't belong in the example file and I think that file is named with a ,py instead of .py) and merge this? Ideally you'd also have a test for this. Or can close it for now



> @andrew8bit want to clean up the example (the last example doesn't belong in the example file and I think that file is named with a ,py instead of .py) and merge this? Ideally you'd also have a test for this. Or can close it for now

wow, this must've been made a long time ago. went ahead and renamed and cleaned example. 
we already have a test for this in the downloader, iirc, this PR is purely for example
